src/platform/forms/tests/save-in-progress/SaveInProgressErrorPage.unit.spec.jsx

```diff
@@ -110,9 +106,6 @@ describe('<SaveInProgressErrorPage>', () => {
     expect(
       findDOM.querySelector('.usa-button-secondary').textContent,
     ).to.contain('Back');
-    expect(findDOM.querySelector('.usa-button-primary').textContent).to.contain(
-      'Continue your application',
-    );
```

Comment:
Did this test break with the changes? I would've expected this to use the default messaging, and the test should verify that.

Reply:
I added it back.